charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,94 @@
+* Creating all standardized HCA pipelines and documenting them  
+* Reviewing, on-boarding, and potentially creating community pipelines  
+* Reviewing community tools for use in data processing pipelines  
+* Working with the community to provide feedback on metadata standards for raw data in the HCA DCP to provide the required information to enable processing of the raw data  
+* Evaluating and adopting file formats and associated standards for outputs of pipelines, ensuring the data is accessible with standard tools and interoperable with downstream processes  
+
+### Pipeline Benchmarking
+
+* Investigating and identifying data for pipeline benchmarking  
+* Establishing and operating automated practices around scientific testing of pipelines  
+* Establishing standard benchmarking used to evaluate data processing pipelines  
+
+### Pipeline Improvement
+
+* Updating pipelines ran in the HCA DCP for optimization of cost, time, or scientific benchmarks  
+
+### Support
+
+* Performing outreach to users on the usability of data in downstream applications  
+* Performing outreach to developers on pipeline development in the HCA DCP  
+* Working with the metadata team and data consumers to ensure the metadata provided about pipelines and the analysis results meet the needs of downstream users  
+* Creating tools that support the creation of community pipelines - may include software, documentation or support  
+
+## Scientific "guardrails"
+
+* Initial versions of standardized pipelines are reviewed by the AWG.  
+* Changes to standardized pipelines that change the scientific output are reviewed by the AWG, engineering updates that do not change the scientific output are not reviewed by the AWG.  
+* Selection of standardized pipelines to work on are approved by the HCA DCP Governance Group (this does not apply to community pipelines, community pipelines are onboarded as a service to the scientific community). Prioritization of the work to develop selected pipelines is performed by the Data Processing Service team.  
+
+## Out of Scope
+
+* Educational training or user assistance on how to perform data analysis with outputs of pipelines  
+* Recommendations or demonstrations of downstream tool use beyond initial data loading  
+* Novel methodological development is not performed by the data processing team unless it is essential to the pipeline, it does not existent in community tools, and it represents minimal effort to implement  
+* Software engineering 3rd party tools  
+
+## Milestones and Deliverables
+
+__2018:__ Two data processing pipelines in production (Smart-seq2 and 10xv2 human data)  
+__2019:__ Adding at least three more data processing pipelines, including 10xv2 snRNA-seq and mouse pipelines for all current analysis pipelines Aim to have at least one _in situ_ transcriptomics pipeline in this year. Aim for one of these pipelines to be a community pipeline. Improved automated testing and benchmarking of analysis pipelines  
```

Comment:
Unity benchmarking service is a place for the community, including the mint team, to run benchmarking comparisons. The benchmarking comparisons that are run in Unity have to come from somewhere, and the mint team develops them for our pipelines. In addition, we have other testing in mind that could be leveraged by Unity someday in the future but is not currently in their immediate roadmap.

I'd prefer not to have to recharter if our priorities shift throughout the year, re more specific dates.

Reply:
Updated editorial suggestion.

Acknowledging bandwidth for automated testing makes sense because the pipeline generators create the benchmarking pipeline specific to that assay which is then installed and ran in Unity. It is kinda like a very complex checker script. If this feels communicated as a duplication we can focus the language on the benchmarking pipeline development.